app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @recipe.name %></h1>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+
+<h2>Ingredients</h2>
+<ul>
+  <% @ingredients.each do |ingredient| %>
```

Comment:
Here is where instead of creating an `@ingredients` instance variable in your controller, you would just iterate over `@recipe.ingredients.each do`.